examples/github-via-ssh/main.tf

```diff
@@ -4,7 +4,7 @@ terraform {
   required_providers {
     flux = {
       source  = "fluxcd/flux"
-      version = "1.2.3"
+      version = "~> 1.2"
```

Comment:
So every time we release a new minor release we need to update all examples?

Reply:
Thats correct, but this is the recommended way to set required provider versions.

Reply:
I will write some docs for this process after we merge this PR.